.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,10 +16,14 @@ For a PR to be considered, each item in the checklist must be checked.
 <!-- Why is this change required? What problem does it solve? If it fixes an open issue, please link to the issue here and auto-close them via commit messages: https://help.github.com/articles/closing-issues-via-commit-messages. -->
 
 ### Screenshots, videos, or demo, if appropriate
-<!-- To record and share a video: http://recordit.co/ -->
+<!-- If you are an outside contributor, please share a reproduction of your work on CodeSandbox: https://codesandbox.io; our Mineral UI Starter (https://codesandbox.io/s/v410y75m0) may be useful for setup. To record and share a video: http://recordit.co/ -->
 
 https://<your-branch-name>--mineral-ui.netlify.com/
 
+<!-- or -->
+
+https://codesandbox.io/s/<your-sandbox-id>
```

Comment:
I wasn't sure if this is the best way we should ask outside contributors to share; thoughts?

Reply:
I like the idea, but what if instead of calling out "outside contributors", we did something like:

> Please share either a reproduction of your work on CodeSandbox (our Mineral UI Starter https://codesandbox.io/s/v410y75m0 may be useful for setup) or a deploy preview.

Reply:
Great idea, thanks for the rewrite!